packages/react-native/package.json

```diff
@@ -143,6 +143,13 @@
         "android": {},
         "jsSrcsDir": "src"
       },
+      {
+        "name": "FBReactNativeSpec_visionOS",
+        "type": "modules",
+        "ios": {},
+        "android": {},
+        "jsSrcsDir": "src/private/specs/visionos_modules"
+      },
```

Comment:
I needed to generate separate set of Codegen specs for visionOS because the built-in one was overwritten when I used Yarn 4 (with Yarn v1 it was working fine) I'm guessing the issue comes from the order of how node_modules are discoevered by codegen. 

If codegen finds `react-native-visionos` first, then `react-native` generated spec overwrites it later, but if it finds it second then everything is working fine. 

cc: @saadnajmi - you mentioned that rn-macos overwrites some Codegen specs of Alert module - maybe you will also run into this or maybe it's only related to adding new modules

Reply:
is there a way to fix this in yarn or RN codegen so it resolved in a human-observable order, rather than inode creation order(UNIX-based) or something else random on Windows? This is a class of support issue that can devolve into "works fine on my machine" :/

Reply:
Are you using yarn 4 with the node_modules linker? curious

Reply:
Yes! With node_modules linker, I'm not sure why there is a discrepancy between those versions but it kind of makes sense as `@callstack/react-native-visionos` is higher in the node_modules folder than `react-native` (therefore the overwrite). I think creating a separate spec for OOT platform-specific modules is a safer bet